File: RobWork/example/python/ex_frame_to_frame_transforms.py
from sdurw import *

def frameToFrameTransforms(a, b, tree_structure, states):
    fk = FKRange(a, b, tree_structure);
    
    result = Transform3DdVector();
    for state in states:
        result.push_back(fk.get(state));
    return result;

File: RobWork/example/lua/ex_frame-to-frame-transforms.lua
function frameToFrameTransforms(a, b, tree_structure, states)
    fk = FKRange(a, b, tree_structure);

    result = Transform3DdVector()
    for i = 0,states:size()-1 do
        result:push_back(fk:get(states[i]))
    end
    return result
end